src/main/webapp/app/shared/metis/metis-conversation.service.ts

```diff
@@ -102,7 +112,16 @@ export class MetisConversationService implements OnDestroy {
         }
         this.activeConversation = cachedConversation;
         this._activeConversation$.next(this.activeConversation);
-    };
+        this.isCodeOfConductPresented = false;
+        this._isCodeOfConductPresented$.next(this.isCodeOfConductPresented);
+    }
+
+    public setCodeOfConduct() {
```

Comment:
What does this do? Can you add a documentation to this public method?

Reply:
It shows the code of conduct baeeaf3e73d143a98619b323696bcf092d709b20